app/assets/v2/js/pages/bounty_details.js

```diff
@@ -538,7 +548,8 @@ var do_actions = function(result) {
   pull_interest_list(result['pk'], function(is_interested) {
 
     // which actions should we show?
-    var show_start_stop_work = is_still_on_happy_path;
```

Comment:
Can't we change this to  
`is_still_on_happy_path = result['status'] != 'expired' || (can_submit_after_expiration_date && result['status'] == 'expired'` ?